lib/pages/login-page.js

```diff
@@ -22,28 +22,23 @@ export default class LoginPage extends AsyncBaseContainer {
 		const passwordSelector = By.css( '#password' );
 
 		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
-		await this.driver.sleep( 1000 );
+		await driverHelper.clickWhenFocused( driver, userNameSelector );
```

Comment:
Implementing a new `driverHelper` method `clickWhenFocused` to address the sleep statement that was previously here.